.github/workflows/wasm.yml

```diff
@@ -141,7 +141,7 @@ jobs:
       - name: Export cache from nix store
         if: ${{ steps.cache.outputs.cache-hit != 'true' && github.event_name != 'merge_group' }}
         run: |
-          nix copy --to "file://${{ env.CACHED_PATH }}?compression=zstd&parallel-compression=true" .#wasm-cargo-artifacts
+          nix copy --to "file://${{ env.CACHED_PATH }}?compression=zstd&parallel-compression=true" .#noir-wasm-cargo-artifacts
```

Comment:
"noir-wasm" is `noir_wasm` and "wasm" is the wasmer backend.

You'll also need to add `noir-wasm-cargo-artifacts` to the list of packages in the flake because it looks like it was never added:
```diff
packages = {
        default = noir-native;

        inherit noir-native;
        inherit noir-wasm;

        # We expose the `*-cargo-artifacts` derivations so we can cache our cargo dependencies in CI
        inherit native-cargo-artifacts;
        inherit wasm-cargo-artifacts;
+ // Add it here
      };
```